packages/website/src/Home.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export const Home = () => (
+  <div>
+    <h2>What is Pocket 'Ponics?</h2>
+    <p>Pocket 'Ponics is a semi-automated hydroponic greenhouse, with an integrated app that allows users to grow fresh fruits and vegetables from the comfort of their own home. With no technical knowledge or agricultural experience, users can set up the greenhouse in a few easy steps, and then allow our automated servers to take care of all the finicky details, like nutrient and water levels. The Pocket 'Ponics app will alert users whenever water needs to be refilled or plants are ready to harvest, so users can enjoy all the benefits of a personal vegetable garden without having to worry about keeping track of it!</p>
+    <h2>Why we built Pocket 'Ponics</h2>
+    <p>By providing easy to use hydroponics that can be grown in a very small space, Pocket 'Ponics hopes to alleviate some of the problems of food insecurity. Because a single greenhouse takes up about the same amount of space as a bookshelf, Pocket 'Ponics allows food to be grown in small spaces such as apartments. As a low-cost, low-effort solution to hydroponic growing, Pocket 'Ponics could provide food desserts with much needed vegetables and fruits, all grown within the comfort of people's homes.</p>
```

Comment:
These could probably be static html files; theres no need to dynaically render static content